server.go

```diff
@@ -723,7 +704,9 @@ func (sp *serverPeer) pushAddrV2Msg(addresses []*addrmgr.NetAddress) {
 	addrs := make([]*wire.NetAddressV2, 0, len(addresses))
 	for _, addr := range addresses {
 		if !sp.addressKnown(addr) {
-			addrV2 := addrmgrToWireNetAddressV2(addr)
+			wireNetAddrType := wire.NetAddressType(addr.Type)
```

Comment:
Calling this out again about the cast since I see the function I commented on in an earlier commit is removed.